src/net.c

```diff
@@ -1,3 +1,8 @@
+/** @file net.c
```

Comment:
See my note above on file headings. We should keep things consistent. Is this format for doxygen?

Reply:
Fixed.